DEVEL.rst

```diff
@@ -8,132 +8,176 @@ If something does not work as expected, please create an issue or
+   $ . devel/bin/activate
+
 
 Continue executing the following commands in the virtualenv you just created.
 
 
-## Install Commissaire
-```
-cd commissaire
-pip install -e .
-popd
-```
+Install Commissaire
+===================
 
+.. code-block::
 
-### Install Commissaire Service
-```
-cd commissaire-service
-pip install -e .
-tools/etcd_init.sh
-```
+   $ cd commissaire
+   $ pip install -e .
+   $ popd
 
-Edit your storage configuration to point to your etcd instance
-```
-cp conf/storage.conf mystorage.conf
-```
-Note: point server_url to http://127.0.0.1:2379 (not https)
+
+Install Commissaire Service
+---------------------------
+
+.. code-block::
+
+   $ cd commissaire-service
+   $ pip install -e .
+   $ tools/etcd_init.sh
+
+
+Edit your storage configuration to point to your etcd instance:
+
+.. code-block::
+
+   $ cp conf/storage.conf mystorage.conf
+
+
+**Note**: Point ``server_url`` to ``http://127.0.0.1:2379`` (not https)
 
 
 Start the service
-```
-commissaire-storage-service -c mystorage.conf &
-popd
-```
 
-### Install Commissaire Server
-```
-cd commissaire-http
-pip install -e .  # Install commissaire-http into the virtualenv
-```
+.. code-block::
+
+   $ commissaire-storage-service -c mystorage.conf &
+   $ popd
+
+
+Install Commissaire Server
+--------------------------
+
+.. code-block::
+
+   $ cd commissaire-http
+   $ pip install -e .  # Install commissaire-http into the virtualenv
+
 
 Edit the configuration to point to your redis instance
-```
-cp conf/commissaire.conf config.conf
-```
-Note: if locally installed you do not need to change anything
 
-Start the service
-```
-commissaire-server -c config.conf &
-popd
-```
+.. code-block::
+
+   $ cp conf/commissaire.conf config.conf
+
+**Note:** If locally installed you do not need to change anything
+
+Start the service:
+
+.. code-block::
+
+   $ commissaire-server -c config.conf &
+   $ popd
+
+
+Run testcases for Commissaire Server
+++++++++++++++++++++++++++++++++++++
+
 
-#### Run testcases for Commissaire Server
-Note that you can use `tox` to run testcases for this project.
+**Note**: that you can use ``tox`` to run testcases for this project.
 
-Install using
+Install using:
 
-```
-pip install tox
-```
+.. code-block::
 
-and then, from the `commissaire-http` folder, run the following
+   $ pip install tox
+
+
+and then, from the ``commissaire-http`` folder, run the following
 command:
 
-```
-tox -v -e py35
-```
+.. code-block:: shell
+
+   $ tox -v -e py35
+
+
+Verification
+------------
+
 
+After this the API will be available at ``http://127.0.0.1:8000/``. To
+verify it works, we will use the initial user ``a`` with pass ``a``.
 
-## Verification
-After this the API will be available at `http://127.0.0.1:8000/`. To
-verify it works, we will use the initial user `a` with pass `a`.
+.. code-block::
 
-```
-curl -u "a:a" -X GET http://127.0.0.1:8000/api/v0/clusters/
-```
+   $ curl -u "a:a" -X GET http://127.0.0.1:8000/api/v0/clusters/
```

Comment:
I believe you are correct. Those keys won't exist until the ``storage-service`` connects for the first time. Until that occurs data can not be transferred on the bus. After you try the workaround above let's see if this continues.